.changeset/config.json

```diff
@@ -3,7 +3,9 @@
   "changelog": "@changesets/cli/changelog",
   "commit": false,
   "fixed": [],
-  "linked": [],
+  "linked": [
```

Comment:
If the app package does not get updated as expected, then switch to fixed packages here